woocommerce/woocommerce
# Update WC Blocks documentation paths

### Changes proposed in this Pull Request:

Updates the WC Blocks documentation in the `hooks` directory, fixes some links that weren't working anymore since the move to the monorepo, and also fixes some lint issues.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Click the links in each document I updated and make sure they resolve.
2. Check the changes made to fix lint errors and make sure they haven't resulted in an undesirable front-end result.



### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
